src/rsa.rs

```diff
@@ -119,6 +119,27 @@ fn verify(padding: &RSAPadding, msg: untrusted::Input,
 }
 
 
+pub mod primitive {
+    use untrusted;
+    /// Lower-level verification function for RSA signatures.
+    ///
+    /// If you do not have the public key available as DER-encoded
+    /// PKCS#1-formatted ASN.1, this function can be used to pass in the raw
+    /// bytes for the public key components as `untrusted::Input` arguments.
+    ///
+    /// Takes an `RSAPadding` static to determine the padding algorithm used,
+    /// a message, a signature (whose length should match the length of public
+    /// exponent, `n`) and a tuple containing the parts of the public key (in
+    /// big-endian byte order). The key length must be between 2048 and 8192 bits.
+    pub fn verify(padding: &super::RSAPadding, msg: untrusted::Input,
+                  signature: untrusted::Input,
+                  (n, e): (untrusted::Input, untrusted::Input))
+                  -> Result<(), ()> {
+        super::verify(padding, msg, signature, (n, e), 2048)
```

Comment:
So we just have `RSAParameters` that carries a reference to an `RSAPadding`, and then a `SignatureVerificationAlgorithm` that carries a reference to an `RSAParameters`? For now, `RSAParameters` can also carry `min_bits`, and then you could add more stuff later.